### ISSUE TITLE
Accessibility : Focus not managed when new screens load

### ISSUE THREAD

**Issue by HarshalCapgemini** [xamarin/Xamarin.Forms#14852]
Accessibility for Android

Focus is not managed when new screens load. Focus remains on hidden content under the screen.
Move focus into the new content that is displayed.

Cannot find any property to set focus in Xamarin forms when new screen loads.

Related to https://github.com/xamarin/Xamarin.Forms/issues/9999

**Comment by Muthukumaradappt** [comment]
Hi @HarshalCapgemini, Please let me know if you find any solution for this.

**Comment by Muthukumaradappt** [comment]
Hi @rachelkang, If I'm coming to the first-time to the page, then I have to focus on the particular element. when on appearing the page like see this example

 protected async override void OnAppearing()
{
            base.OnAppearing();
            Xamarin.CommunityToolkit.Extensions.SemanticExtensions.SetSemanticFocus(headerText); //Label
 #}

But it will throw the Exception => Cannot access the view

**Comment by jfversluis** [comment]
No response for a long time, assuming the author has resolved this.
